### PR TITLE
:seedling: logs info updated

### PR DESCRIPTION
**Pull request by Sajiyah-Salat**
There was same logs at different place which results in difficulty to debug. 
Updated the unique logs to give better understanding.
Partially fixes: #3435

PS: I am moving forward with this #3506 pr.

### CODE REVIEW

**Review by Kavinjsir — APPROVED** [review]
lgtm 👍🏼

**Review by camilamacedo86 — CHANGES_REQUESTED** [review]
@Sajiyah-Salat 

Can you please squash the commits for we are able to move forward?

File: hack/docs/internal/cronjob-tutorial/generate_cronjob.go [hunk 1]
```diff
@@ -553,7 +553,7 @@ var _ = AfterSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 })
 `, SuiteTestCleanup)
-	CheckError("fixing suite_test.go", err)
+	CheckError("updating suite_test.go to test cleanup", err)
```
Comment (varshaprasad96) [inline hack/docs/internal/cronjob-tutorial/generate_cronjob.go:556]: Nit:
```suggestion
	CheckError("updating suite_test.go to cleanup tests", err)
```

**Review by varshaprasad96 — COMMENTED** [review]
Just a nit, looks good after squashing the commits!
Nice work @Sajiyah-Salat

**Comment by Sajiyah-Salat** [comment]
Hello @camilamacedo86 Just a small request. I have squash the commits. If still there is any changes require let me know we will start a new pr for that. Just merge this one. As yo know I have squashing issues a lot.

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

/approved

/hold cancel

**Review by camilamacedo86 — COMMENTED** [review]
Moving forward with this one since it is blocked by the preview of docs which is an issue that we are sorting out in another PR and has no relation with this one.

**Pull request merged**